.github/workflows/main.yml

```diff
@@ -387,7 +387,7 @@ jobs:
     - name: Check out the Unbundled AAOS Repo
       run: |
         HOST=https://android.googlesource.com/platform/manifest
-        BRANCH=ub-automotive-master-20231102
+        BRANCH=ub-automotive-master-20240521
         mkdir "$ORG_GRADLE_PROJECT_unbundledAAOSDir"
         cd "$ORG_GRADLE_PROJECT_unbundledAAOSDir"
         repo init -u $HOST -b $BRANCH -g pdk,pdk-fs --depth=1
```

Comment:
@timothyfroehlich This filters out the prebuilts/sdk/. Do you know how to modify this properly to include that?